web/README.md

```diff
@@ -0,0 +1,32 @@
+## AirQo Website 
+https://airqo.net
+
+### Local Setup Using Docker 
+The AirQo website can be configured locally for development and testing purposes.
+#### **Requirements**
+- [Docker](https://www.docker.com/)
+- Download and extract [CodeIgniter](https://codeigniter.com/) System folder at the root of this directory. Link: https://api.github.com/repos/bcit-ci/CodeIgniter/zipball/refs/tags/3.1.11
+- Open ports: 8000, 8001, 88036
+
+#### **Setup**
+
+Checkout this branch locally and `cd` to `AirQo-frontend/web`.  Get the `.env` from the PR author. Also add the database dump (e.g. airqo_db.sql) in `db_data` folder.
```

Comment:
`Get the .env from the PR author` lol... We will be coming to you in the future 😃

Reply:
lol.. .env was shared in our multi-DM

Reply:
@danielogen I managed to get the .env and the sql file. I cant access the app when I run docker-compose up

Reply:
@ja-odur, do you have the magical system folder????

Reply:
@NoahNsimbe I downloaded that too. added it to the root folder (web)